cardano-testnet/src/Testnet/Process/Cli/SPO.hs

```diff
@@ -52,10 +52,10 @@ checkStakePoolRegistered
   :: (MonadTest m, MonadCatch m, MonadIO m, HasCallStack)
   => TmpAbsolutePath
   -> ExecConfig
-  -> FilePath -- ^ Stake pool cold verification key file
+  -> File (VKey StakeKey) In -- ^ Stake pool cold verification key file
```

Comment:
I think it would be helpful to include the fact that this is a cold key in the type.

Reply:
Oh, that's a typo. I've used `StakePoolKey` type instead in https://github.com/IntersectMBO/cardano-node/pull/6026